packages/realm-server/server.ts

```diff
@@ -336,15 +338,25 @@ export class RealmServer {
       ...(backgroundURL ? { backgroundURL } : {}),
     });
     if (this.seedPath) {
-      let ignoreList = IGNORE_SEED_FILES.map((file) =>
-        join(this.seedPath!.replace(/\/$/, ''), file),
-      );
-      copySync(this.seedPath, realmPath, {
-        filter: (src, _dest) => {
-          return !ignoreList.includes(src);
-        },
-      });
-      this.log.debug(`seed files for new realm ${url} copied to ${realmPath}`);
+      if (copyFromSeedRealm) {
+        let ignoreList = IGNORE_SEED_FILES.map((file) =>
+          join(this.seedPath!.replace(/\/$/, ''), file),
+        );
+
+        copySync(this.seedPath, realmPath, {
+          filter: (src, _dest) => {
+            return !ignoreList.includes(src);
+          },
+        });
+        this.log.debug(
+          `seed files for new realm ${url} copied to ${realmPath}`,
+        );
+      } else {
+        copySync(
+          join(this.seedPath!.replace(/\/$/, ''), 'index.json'),
+          join(realmPath!.replace(/\/$/, ''), 'index.json'),
+        );
```

Comment:
good! i was worried about this, glad you noticed it

Reply:
I wonder if it's simpler to just use `writeJSONSync` to manually set this file--it's really small.
```suggest
writeJSONSync(join(realmPath, 'index.json'), {
  "data": {
    "type": "card",
    "meta": {
      "adoptsFrom": {
        "module": "https://cardstack.com/base/cards-grid",
        "name": "CardsGrid"
      }
    }
  }
});
```

Reply:
(also, I thought the `join` took care of the trailing `/`, look at how we set the `.realm.json` above.)